.github/workflows/all-green.yml

```diff
@@ -0,0 +1,26 @@
+name: Check Pull Request CI Status
+
+on: # yamllint disable-line rule:truthy
+  pull_request:
+    types:
```

Comment:
- The listed types are the default values to be removed. 
- Scoping to master branch would be ideal.